package.js

```diff
@@ -29,5 +29,8 @@ Package.onUse(function(api) {
   ], 'client');
 
   if (api.export)
-    api.export('Wizard', 'client');
+    api.export([
+      'Wizard',
+      'WizardRouter'
```

Comment:
Not problem. This change was there before AutoForm update commit. Going to fix now!